javascript/lib/api/src/client/request-factory/request-sender.ts

```diff
@@ -51,6 +51,19 @@ export abstract class RequestSender {
       });
   }
 
+  public fetchMergeRequest<T>(options: ParseRequest<T>): Promise<MergeResponse<T>> {
+    return this.fetchRequest(options).then(response => {
+      if (response.status === 204) {
+        return new MergeResponse(
+          options.parser(response.body),
```

Comment:
For status 204 (no content) there is no content in the response, so parsing it is not required, you can pass in `null` (same as for PUT).

Reply:
Went back to using PutResponse